### PR TITLE
Redesign features on Product Details Page

### PR DESCRIPTION
**Pull request by Treesa-James**
Added Market Intel Details and Note Form

### CODE REVIEW

File: src/app/productdetails/productdetails.component.ts [hunk 1]
```diff
@@ -167,6 +201,8 @@ export class ProductdetailsComponent implements OnInit {
             this.interStateQuantity = this.interStateQuantity + branch.qty;
           }
         });
+        console.log("Notes :",this.productDetail.notes);
```
Comment (arun-experion) [inline src/app/productdetails/productdetails.component.ts:204]: remove this unwanted code

**Review by arun-experion — COMMENTED** [review]
code review comment added

File: src/app/productdetails/productdetails.component.ts (hunk 1 above)
Comment (Treesa-James) [reply src/app/productdetails/productdetails.component.ts]: corrected

**Review by arun-experion — COMMENTED** [review]
code review completed

**Pull request merged**